Doc/library/inspect.rst

```diff
@@ -153,6 +153,19 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |                 | f_trace           | tracing function for this |
 |                 |                   | frame, or ``None``        |
 +-----------------+-------------------+---------------------------+
+|                 | f_trace_lines     | indicate whether a        |
+|                 |                   | tracing event is          |
+|                 |                   | triggered for each source |
+|                 |                   | source line               |
++-----------------+-------------------+---------------------------+
+|                 | f_trace_opcodes   | indicate whether          |
+|                 |                   | per-opcode events are     |
+|                 |                   | requested                 |
++-----------------+-------------------+---------------------------+
+|                 | clear             | used to clear all         |
+|                 |                   | references to local       |
+|                 |                   | variables                 |
```

Comment:
The docstring says *most*, not *all*, references:

```suggestion
|                 | clear             | used to clear most        |
|                 |                   | references held by the    |
|                 |                   | frame                     |
```

Reply:
I can't find where it's says _most_
From [doc](https://docs.python.org/3.14/reference/datamodel.html#frame.clear):
> This method clears all references to [local variables](https://docs.python.org/3.14/reference/executionmodel.html#naming) held by the frame. Also, if the frame belonged to a [generator](https://docs.python.org/3.14/glossary.html#term-generator), the generator is finalized. This helps break reference cycles involving frame objects (for example when catching an [exception](https://docs.python.org/3.14/library/exceptions.html#bltin-exceptions) and storing its [traceback](https://docs.python.org/3.14/reference/datamodel.html#traceback-objects) for later use).

Reply:
I copied the docstring. If the docstring is outdated, would you mind to also updated it?

Reply:
Will do!